index.html

```diff
@@ -5838,6 +5838,20 @@ <h3>Group Control</h3>
         </p>
       </section>
     </section>
+    <section>
+      <h2>Changing the DID subject</h2>
+      <p>
+A <a>DID document</a> has exactly one <a>DID subject</a>, identified by exactly
```

Comment:
```suggestion
A <a>DID document</a> should have one <a>DID subject</a>, identified by exactly
```

Not sure this is correct to be honest. Just wanting to offer some sort of definitive change.

This text seems to conflict with statements in the persistent section. In particular:

>It is technically possible for the controller to reuse a DID for different subjects and, more subtly, for the precise definition of the Subject to either change over time or be misunderstood. 

>Because of these potential ambiguities, DIDs should be considered valid contextually rather than absolutely. Their persistence does not imply that they refer to the exact same Subject, nor that they are under the control of the same controller. Instead, one needs to understand the context in which the DID was created, how it is used, and consider the likely shifts in their meaning, and adopt procedures and policies to address both potential and inevitable semantic drift. 

From this section I infer that it's not possible and shouldn't be done, but from the persistence section I infer that it's a consideration that's left contextually up to the party relying upon the DID to decide how they'll deal with it.

Reply:
@kdenhartog --- 

I think what's missing from your reading, and perhaps from the writing, is "at a time" --- i.e., `A <a>DID document</a> has exactly one <a>DID subject</a>[ at a time], ...`.  This allows for a DID subject to be a temporal or collective entity, which is identified by a single URI; but does not allow for a DID subject to be a collection of entities, each identified by its own single URI.

The other sections you quoted are not about having multiple URIs which identify multiple subject entities in a single DID document, but about having a single URI which identifies multiple subject entities *over time*.  In egregious pseudo-code, `did:example:xyz did:subject example:orgPresident` could be used to designate the subject of the DID as the President of an organization, which role might shift from one person to another over time.  The `example:orgPresident` is always the subject of that DID, but the person in that role will change periodically, so the effective *meaning* of the DID subject will change.

Perhaps that helps?

Reply:
+1 yeah that's a great nuance that makes them aligned. I'll update my suggestion with your text and approve pending the merging up that change.